adam/batch_run_manager.py

```diff
@@ -216,3 +216,4 @@ def run(self):
         self._submit()
         self._wait_for_completion()
         self._get_results()
+        print("Run status: complete")
```

Comment:
I wonder if it would be clearer to print the current state, i.e. `self.state` ?

Reply:
@carise : That's much nicer. Made the change. PTAL.